tests/test_new_meshing_workflow.py

```diff
@@ -920,3 +921,65 @@ def downstream_names(task):
         "Enclose Fluid Regions (Capping)",
         "Create Regions",
     ]
+
+
+@pytest.mark.skip("Randomly failing in CI")
+@pytest.mark.codegen_required
+@pytest.mark.fluent_version(">=23.2")
+def test_attrs_in_watertight_meshing_workflow(new_mesh_session):
+    # Import geometry
+    import_file_name = examples.download_file(
+        "mixing_elbow.pmdb", "pyfluent/mixing_elbow"
+    )
+    watertight = new_mesh_session.watertight()
+    unwanted_attrs = {"fault_tolerant", "part_management", "pm_file_management"}
+    assert set(dir(watertight)) - unwanted_attrs == set(dir(watertight))
+
+    for attr in unwanted_attrs:
+        with pytest.raises(AttributeError) as msg:
+            getattr(watertight, attr)
+        assert (
+            msg.value.args[0]
+            == f"'WatertightMeshingWorkflow' object has no attribute '{attr}'"
+        )
+
+    watertight.import_geometry.file_name.set_state(import_file_name)
+    watertight.import_geometry.length_unit = "in"
+    watertight.import_geometry()
+
+    assert watertight.import_geometry.file_name()
+    # Resets the workflow:
+    watertight.watertight()
```

Comment:
This should be via an explict reset command.

Reply:
yes, sounds good. I also need to understand some concerns with reseting in the meeting today.

Reply:
Done.